src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -190,6 +190,25 @@ void overrideLocationAndInstanceId() throws IOException {
         workspaceCreator.email.toLowerCase(),
         describeResource.privateUserName.toLowerCase(),
         "describe output matches private user name");
+
+    // new key-value pair will be appended, existing key-value pair will be updated.
+    String newName = "NewOverrideLocationAndInstanceId";
+    String newDescription = "\"new override default location and instance id\"";
+    String newMetadata = "NewMetadata=metadata";
+    UFGcpNotebook updatedNotebook =
+        TestCommand.runAndParseCommandExpectSuccess(
+            UFGcpNotebook.class,
+            "resource",
+            "update",
+            "gcp-notebook",
+            "--name=" + name,
+            "--new-name=" + newName,
+            "--new-description=" + newDescription,
+            "--new-metadata=" + newMetadata);
+
+    // check that the properties match
+    assertEquals(newName, updatedNotebook.name, "create output matches name");
+    assertEquals(newDescription, updatedNotebook.description, "create output matches description");
```

Comment:
We need a test case where you update more than one metadata entry. Also put a comment to say that we can't assert on the metadata because it's not stored in or accessible via Workspace Manager.

Reply:
you can get it in the instance. for example, we are getting some of the instance fields here. https://github.com/DataBiosphere/terra-cli/blob/1573d96dad78799f8c8168aecea1311c73537511/src/main/java/bio/terra/cli/serialization/userfacing/resource/UFGcpNotebook.java#L36 it could be a followup as you are adding the metadata to the cli in a separate ticket .

Reply:
We'd need to add it to `UFGcpNotebook` to get it to display in the text and JSON outputs.

Reply:
Thanks Yu and Jay. In this ticket, I add the `split = ","` and support multiple entries at this ticket. 
And the display after the command with the test will do in the next pr.